proxy/server.dev-host-macos.conf

```diff
@@ -7,9 +7,9 @@ server {
     error_page 500 502 503 504 /50x.html;
 
     location /api/ {
-#        proxy_pass https://server-radix-api-dev.dev.radix.equinor.com; #for k8s hosted API
+       proxy_pass https://server-radix-api-qa.dev.radix.equinor.com; #for k8s hosted API
         #for localhost API on MacOS https://docs.docker.com/desktop/networking/#i-want-to-connect-from-a-container-to-a-service-on-the-host
-        proxy_pass http://host.docker.internal:3002;
+#        proxy_pass http://host.docker.internal:3002;
```

Comment:
are these changes intended? if so ignore my comment

Reply:
This was invalid link